internal/ls/definition.go

```diff
@@ -15,25 +16,70 @@ func (l *LanguageService) ProvideDefinitions(fileName string, position int) []Lo
 	}
 
 	checker := program.GetTypeChecker()
+	calledDeclaration := tryGetSignatureDeclaration(checker, node)
```

Comment:
I tried to port the logic from `src/services/goToDefinition.ts` in the TypeScript repository.